src/js/components/FilterButtons.js

```diff
@@ -36,8 +36,8 @@ class FilterButtons extends React.Component {
     let filterCount = this.getCount(itemList);
 
     return filters.map((filter) => {
-      let classSet = classNames({
-        'button button-stroke button-inverse': true,
+      let classSet = classNames('button button-stroke',{
```

Comment:
can we add another `prop` to this component called `className` and have it default to `button button-stroke` and then use it here? that way, we can have the buttons not stroke if don't want it to :)